.github/workflows/release.yml

```
@@ -41,14 +49,27 @@ jobs:
       - name: "Prepare the release note"
         run: |
           shasha=$(shasum -a 256 _artifacts/SHA256SUMS | awk '{print $1}')
+          version="VERSION"
+          [[ $GITHUB_REF == v* ]] && version="${GITHUB_REF##*/v}"
```

Doesn't this need to match the `ref/tags` part too?

```suggestion
          [[ $GITHUB_REF == refs/tags/v* ]] && version="${GITHUB_REF#refs/tags/v}"
```

---

Thanks, fixed